package.json

```diff
@@ -33,7 +33,8 @@
     "http": "static spec/jasmine -H '{\"Cache-Control\": \"no-cache, must-revalidate\"}'",
     "****** NODE *****": "",
     "postinstall": "cd docs && yarn",
-    "precommit": "lint-staged & grunt exec:docs_dist & wait && git add -A docs/{index.html,dist/**/*}"
+    "precommit": "lint-staged & grunt exec:docs_dist",
```

Comment:
this was my attempt at running these tasks in parallel, then waiting for both tasks to finish and doing the next one.
Just roll it back to
```
"precommit": "lint-staged && grunt exec:docs_dist && git add -A docs/{index.html,dist/**/*}"
```